Busybox fails to build with glibc-2.16.
loginutils/passwd.c:104:16: error: storage size of 'rlimit_fsize' isn't known
loginutils/passwd.c:188:12: error: 'RLIMIT_FSIZE' undeclared (first use in this function)

Index: busybox-1.18.5/include/libbb.h
===================================================================
--- busybox-1.18.5.orig/include/libbb.h	2012-10-21 15:26:21.000000000 +0200
+++ busybox-1.18.5/include/libbb.h	2012-10-21 15:26:47.000000000 +0200
@@ -29,6 +29,7 @@
 #include <sys/poll.h>
 #include <sys/ioctl.h>
 #include <sys/mman.h>
+#include <sys/resource.h>
 #include <sys/socket.h>
 #include <sys/stat.h>
 #include <sys/time.h>
